frontend/model/contracts/group.js

```diff
@@ -368,9 +422,18 @@ sbp('chelonia/defineContract', {
             const toUser = vueFetchInitKV(fromUser, data.toUser, [])
             toUser.push(data.paymentHash)
           }
-          paymentMonth.lastAdjustedDistribution = groupIncomeDistribution({
-            state, getters, updateMonthstamp, adjusted: true
+          console.error(payment)
```

Comment:
This shouldn't be here...

<img width="1224" alt="Screen Shot 2021-11-27 at 11 35 44 AM" src="https://user-images.githubusercontent.com/138706/143719975-3c59018e-aa3d-46c2-ab84-8593831b2962.png">

Reply:
Nice catch.

Fixed.